tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -0,0 +1,68 @@
+import cv2
+import numpy as np
+import torch
+import torchvision
+from torchvision.models._utils import IntermediateLayerGetter
+
+
+class DFBR:
+    r"""Deep Feature based Registration
+
+    This class implements a CNN feature based registration,
+    as proposed in a paper titled `Deep Feature based Cross-slide Registration
+    <https://arxiv.org/pdf/2202.09971.pdf>`_.
```

Comment:
Are you able to add any metrics?

Reply:
What do you mean by metrics here? Can you please clarify?

Reply:
Accuracy, RMSE, TRE, ...

Reply:
These metrics are not needed at this point for this PR. However, they can be computed against the manual landmark points once we have computed the transformation parameters. I will add a separate function for metrics later.

Reply:
Let's add these metrics once you have the final results.